public/app/features/dashboard/components/SubMenu/DashboardLinks.tsx

```diff
@@ -21,11 +21,8 @@ export const DashboardLinks: FC<Props> = ({ dashboard, links }) => {
   const [, forceUpdate] = useReducer((x) => x + 1, 0);
```

Comment:
nit: Not part of this PR, but I think I've seen an `useForceUpdate` hook somewhere

Reply:
ah, good catch updated to useForceUpdate